web/src/test/groovy/org/apache/shiro/web/config/IniFilterChainResolverFactoryTest.groovy

```diff
@@ -41,6 +42,7 @@ import static org.hamcrest.MatcherAssert.assertThat
  *
  * @since 1.2
  */
+@Isolated
```

Comment:
nit: add reason

My guess is that the INI / reflection builder _might_ be sys prop related?